.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data requested**
+A clear and concise description of the data needed
+
+**How would this data be visualized?**
+- [ ] Histogram
+- [ ] Pie Chart
+- [ ] Column Chart
+- [ ] Grouped Column Chart
+- [ ] Bar Chart
+- [ ] Grouped Bar Chart
+- [ ] Line Graph
+
+
+**Which countries is this data related to?**
+- [ ] Burkina Faso
+- [ ] DRC Congo
+- [ ] Ethiopia
+- [ ] Kenya
+- [ ] Nigeria
+- [ ] Senegal
+- [ ] South Africa
+- [ ] Tanzania
+- [ ] Uganda
+- [ ] Zambia
```

Comment:
Let's change this to a prose format i.e Burkina Faso, DRC Congo, etc. A non-technical user wouldn't know how to use this when creating the data requests.

This also goes for the rest of the lists.